sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -65,7 +65,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         var category = new Sourcemash.Models.Category({ id: id });
         category = this._categories.add(category);
 
-        var categoryItems = new Sourcemash.Collections.Items([], {category: category});
+        var categoryItems = new Sourcemash.Collections.Items([], {category: category, allItems: this._user.get('email') ? false : true});
```

Comment:
I went to a categories page once when logged in and not subscribed to anything, and once when not. I made the change so that only one item showed up when I was logged in, and three when not.